mdata/cache/accnt/accnt_test.go

```diff
@@ -0,0 +1,128 @@
+package accnt
+
+import (
+	"testing"
+	"time"
+)
+
+func getNew(maxSize int) *FlatAccnt {
+	a := &FlatAccnt{
+		total:       0,
+		metrics:     make(map[string]*FlatAccntMet),
+		maxSize:     uint64(maxSize),
+		lru:         NewLRU(),
+		evictQ:      make(chan *EvictTarget, 10),
```

Comment:
yeah constructors should return their concrete types. As far as I know that's the standard way to do it in go.
you can then assign the output of the constructor to interface types. I don't think you need `NewAccount`